tokio-codec/README.md

```diff
@@ -0,0 +1,37 @@
+# tokio-io
+
+Core I/O abstractions for the Tokio stack.
```

Comment:
The description probably should be something like what you have in the `Cargo.toml`: "Utilities for encoding and decoding frames."`.

Then elaborate a bit about how these utilities go from byte streams (`AsyncRead` / `AsyncWrite`) and turn them into `Sink + `Stream` instances that work at the frame level, where a frame is a discrete protocol unit. Frames are protocol specific, for example in a line based protocol, the frame is the line. In HTTP, the frame could be the entire HTTP header block. In HTTP/2.0, the frame is length delimited with a fixed size head [link](https://httpwg.org/specs/rfc7540.html#FramingLayer).